index.pug

```diff
@@ -0,0 +1,591 @@
+head
+  meta(http-equiv='Content-Type' content='text/html; charset=utf-8')
+  meta(http-equiv='Content-Type' content='text/html; charset=utf-8')
+  meta(name='viewport' content='width=device-width, initial-scale=1.0')
+  link(rel='shortcut icon' href='images/favicon.png' type='image/png')
+  title Razorpay - Best Payment Gateway for Online Payments - India
+  link(href='css/style.css' rel='stylesheet' type='text/css')
+  link(href='css/common.css' rel='stylesheet' type='text/css')
+// section 1 start
```

Comment:
this is an invalid header please follow along with the guidelines, but I'm attaching the sample below,

also please follow the intention properly
![Screenshot 2021-12-03 at 7 14 06 PM](https://user-images.githubusercontent.com/10862593/144612415-7e5dcf73-0063-490a-80f8-a37ee10556e5.png)